src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/cli/usecases/showemails/ShowEmailContentCLIPrompt.java

```diff
@@ -18,24 +17,18 @@ public class ShowEmailContentCLIPrompt extends AuthorizedCLIPrompt {
 
   private final ProvideEmailsUseCase provideEmailsUseCase;
 
-  private final MailboxType mailboxType;
-
   public ShowEmailContentCLIPrompt(
       BaseCLIPrompt previousPrompt,
-      Email email,
-      ProvideEmailsUseCase provideEmailsUseCase,
-      AuthUseCase authUseCase,
-      MailboxType mailboxType) {
+      Email email, ProvideEmailsUseCase provideEmailsUseCase, AuthUseCase authUseCase) {
```

Comment:
I could imagine that it isnt formatted correctly

Reply:
You are right!